hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/OzoneTestUtils.java

```diff
@@ -92,7 +92,7 @@ public static void closeContainers(
             .updateContainerState(ContainerID.valueOf(blockID.getContainerID()),
                 HddsProtos.LifeCycleEvent.CLOSE);
       }
-      Assert.assertFalse(scm.getContainerManager()
+      Assertions.assertFalse(scm.getContainerManager()
```

Comment:
Nit: Can we import the static methods itself like in other test changes?

Reply:
I agree it would be good to consistently add static imports for assertions to make code more readable.  Let's do it in follow-up task, since it's another big, but straightforward change, and also affects most other sub-modules.

Reply:
I acknowledge,